microcosm_pubsub/registry.py

```diff
@@ -94,7 +95,7 @@ def find(self, media_type):
         try:
             # use a concrete schema class if any
             schema_cls = self._mappings[matching_media_type]
-            schema = schema_cls()
+            schema = schema_cls(unknown=EXCLUDE)
```

Comment:
The `unknown` instance variable is used as default when no value is passed in directly, which is typically the case in the serialization [`validate`](https://github.com/globality-corp/microcosm-pubsub/blob/develop/microcosm_pubsub/codecs.py#L85) call (for which `unknown` cannot be passed in as an argument).